toolkits/dropbox/evals/eval_download_file.py

```diff
@@ -0,0 +1,89 @@
+from arcade.sdk import ToolCatalog
+from arcade.sdk.eval import (
+    BinaryCritic,
+    EvalRubric,
+    EvalSuite,
+    ExpectedToolCall,
+    NoneCritic,
+    tool_eval,
+)
+
+import arcade_dropbox
+from arcade_dropbox.critics import DropboxPathCritic
+from arcade_dropbox.tools.files import download_file
+
+rubric = EvalRubric(
+    fail_threshold=0.8,
+    warn_threshold=0.9,
+)
+
+
+catalog = ToolCatalog()
+catalog.add_module(arcade_dropbox)
+
+
+@tool_eval()
+def download_file_eval_suite() -> EvalSuite:
+    """Create an evaluation suite for the download_file tool."""
+    suite = EvalSuite(
+        name="download_file",
+        system_message="You are an AI assistant that can interact with files and folders in Dropbox using the provided tools.",
+        catalog=catalog,
+        rubric=rubric,
+    )
+
+    suite.add_case(
+        name="Download file in the root folder by file path",
+        user_message="Download the file test.txt from Dropbox",
+        expected_tool_calls=[
+            ExpectedToolCall(
+                func=download_file,
+                args={
+                    "file_path": "test.txt",
+                    "file_id": None,
+                },
+            ),
+        ],
+        critics=[
+            DropboxPathCritic(critic_field="file_path", weight=0.5),
+            NoneCritic(critic_field="file_id", weight=0.5),
```

Comment:
as discussed offline, these should not use NoneCritic if you want them to be critisized